code/modules/antagonists/fugitive/fugitive.dm

```diff
@@ -0,0 +1,88 @@
+
+
+/datum/antagonist/fugitive
+	name = "Fugitive"
+	//show_in_antagpanel = FALSE //remove this later- they are event specific. this is 100% for testing
+	roundend_category = "Fugitive"
+	silent = TRUE //greet called by the event
+	var/datum/team/fugitive/fugitive_team
+	var/is_captured = FALSE
+
+/datum/antagonist/fugitive/apply_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	update_fugitive_icons_added(M)
+
+/datum/antagonist/fugitive/remove_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	update_fugitive_icons_removed(M)
+
+/datum/antagonist/fugitive/greet(backstory)
+	to_chat(owner, "<span class='boldannounce'>You are the Fugitive!</span>")
+	switch(backstory)
+		if("prisoner")
```

Comment:
*cries oop tears*
Also direct sleeps in main add_antag chain are not recommended.

Reply:
i was hoping it wouldn't freeze the proc that called it since greet doesn't return anything and is also optional for some antagonists, but that's what sleep does. is the only solution changing the backstory or putting it all into multiple procs on callback timers? because i'd want to avoid the proc solution

Reply:
It's just some outputs, parameterless INVOKE_ASYNC is enough here.

Reply:
okay, thanks